users/vieting/experiments/switchboard/transducer/feat/experiments.py

```diff
@@ -63,6 +73,12 @@ def get_ctc_alignment() -> List[tk.Path]:
         },
         "rtf": 5,
     }
+    if ctc_alignment_model == "scf":
+        align_args["train_exp_name"] = "conformer_bs2x5k_scf_baseline_preemphasis97_wn"
+    elif ctc_alignment_model == "mel":
+        align_args["train_exp_name"] = "conformer_bs5k_lgm80_baseline"
+    else:
+        raise ValueError(f"Unknown ctc_alignment_model: {ctc_alignment_model}")
```

Comment:
I would make this configurable and then infer which ctc system to run from that. E.g.
```
input: ctc_alignment_model = "conformer_bs2x5k_scf_baseline_preemphasis97_wn"
if "scf" in ctc_alignment_model:
    _, ctc_nn_system = run_scf_baseline_ctc()
...
align_args = {
    ...
    "train_exp_name": ctc_alignment_model,
    ...
}
```

Reply:
That makes sense.